Automattic/jetpack
# AI Assistant: Register ai-assistant-backend-prompts beta extension

Part of #28569.

## Proposed changes:

* Register `ai-assistant-backend-prompts` as a beta extension so we can test the backend prompts on a smaller beta before releasing it

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Test in Simple, Atomic, and Simple sites
* Go to the block editor
* Open the dev console
* With the `beta extensions` disabled, confirm that `Jetpack_Editor_Initial_State.available_blocks['ai-assistant-backend-prompts']` is not set:

```
Jetpack_Editor_Initial_State.available_blocks['ai-assistant-backend-prompts']
```

<img width="500" alt="Screenshot 2023-09-12 at 17 50 00" src="https://github.com/Automattic/jetpack/assets/6760046/03417a4a-27fa-41e9-aa07-e279da638230">

* With the `beta extensions` enabled, confirm that `Jetpack_Editor_Initial_State.available_blocks['ai-assistant-backend-prompts']` is set to `available: true`:

<img width="500" alt="Screenshot 2023-09-12 at 17 51 23" src="https://github.com/Automattic/jetpack/assets/6760046/ea497dbc-cdbb-41ab-920a-bc94ef727ed0">
